roles/openshift_logging/defaults/main.yml

```diff
@@ -72,7 +72,7 @@ openshift_logging_fluentd_nodeselector: "{{ openshift_hosted_logging_fluentd_nod
 openshift_logging_fluentd_cpu_limit: 100m
 openshift_logging_fluentd_memory_limit: 512Mi
 openshift_logging_fluentd_es_copy: false
-openshift_logging_fluentd_use_journal: "{{ openshift_hosted_logging_use_journal | default('') }}"
+openshift_logging_fluentd_use_journal: "{{ openshift_hosted_logging_use_journal if openshift_hosted_logging_use_journal is defined else (docker_log_driver|default('') == 'journald') | ternary(True, False) }}"
```

Comment:
where is `docker_log_driver` being set? and is that preferable to using the docker option from openshift_facts? 
e.g. {{ openshift.docker.options | search('journald') }}